src/server/pegasus_manual_compact_service.cpp

```diff
@@ -204,8 +214,16 @@ bool pegasus_manual_compact_service::check_periodic_compact(
     }
 
     auto now = static_cast<int64_t>(now_timestamp());
+    int64_t cur_day_midnight = dsn::utils::get_unix_sec_today_midnight();
     for (auto t : trigger_time) {
         auto t_ms = t * 1000;
+
+        if (_manual_compact_first_day_s.load() == cur_day_midnight && t_ms < now &&
```

Comment:
It's allowed to set multiple periodic compact tasks on a table, for example there are 2 tasks set to be executed at 02:00 and 16:00, now real world time is 15:00, will the second task (16:00 one) to be executed normally?